containers/unit_tests/TestVector.hpp

```diff
@@ -116,6 +235,10 @@ TEST(TEST_CATEGORY, vector_combination) {
   test_vector_combinations<int, TEST_EXECSPACE>(3057);
 }
 
+TEST(TEST_CATEGORY, vector_insert) {
+  Impl::test_vector_insert<int, TEST_EXECSPACE>(3057);
```

Comment:
Also we should probably test with a scalar of type `size_t` as well; that's likely to lead to issues in SFINAE (if they exist).

Reply:
It exceeds it since i more than double the size.

Reply:
Where? I couldn't find it on a first look-through. It should probably be commented there anyway so that if someone gets an error on that line they'll know why